lib/dl_api_lib/dl_api_lib_tests/db/control_api/test_info.py

```diff
@@ -83,3 +83,11 @@ def test_get_connector_form_bad_request(self, client):
 
         form_resp = client.get(f"/api/v1/info/connectors/forms/{CONNECTION_TYPE_CLICKHOUSE.name}/bad_form_mode")
         assert form_resp.status_code == 400, form_resp.json
+
+    def test_get_connector_icon(self, client):
+        form_resp = client.get("/api/v1/info/connectors/icons/list")
+        assert form_resp.status_code == 200
+        resp = form_resp.json
+        assert "icons" in resp
+        assert "type" in resp["icons"][0]
+        assert resp["icons"][0]["type"] == "data"
```

Comment:
Let's check all icons, that is:
1. Assert that `resp["icons"]` is present and not empty
2. Assert this lines for all items in `resp["icons"]`